javascript/javascript2/week2/Test.js

```diff
@@ -0,0 +1,107 @@
+// Doubling a number
+let numbers = [1, 2, 3, 4];
+let newNumbers = numbers
+  .filter((element) => element % 2 !== 0)
+  .map((article) => article * 2);
+console.log("The doubled numbers are", newNumbers);
+
+const movies = [
+  {
+    title: "'A' gai wak",
+    year: 1983,
+    rating: 7.4,
+    votes: 11942,
+    running_times: 6300,
+  },
+  {
+    title: "'Breaker' Morant",
+    year: 1980,
+    rating: 7.9,
+    votes: 10702,
+    running_times: 6420,
+  },
+  {
+    title: "'Crocodile' Dundee II",
+    year: 1988,
+    rating: 5.5,
+    votes: 47180,
+    running_times: 6480,
+  },
+  {
+    title: "(500) Days of Summer",
+    year: 2009,
+    rating: 7.7,
+    votes: 412368,
+    running_times: 5700,
+  },
+  {
+    title: "*batteries not included",
+    year: 1987,
+    rating: 6.6,
+    votes: 25636,
+    running_times: 6360,
+  },
+];
+// 1 - the movies with a short title
+let arrayOfShortMoviesTitle = movies
```

Comment:
Just for the optimisation instant of calling a filter function 3 times call it once make the condition inside the filter. No need to call separate filter for other conditions.

Reply:
Implemented the feedback changes. Kindly check and approve!